docs/content/docs/try-flink/local_installation.md

```diff
@@ -24,71 +24,134 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Local Installation
- 
-{{< unstable >}}
-{{< hint info >}}
-  NOTE: The Apache Flink community only publishes official builds for
-  released versions of Apache Flink.
+# First steps
+
+<span style="font-size:larger;">Welcome to Flink! :)</span>
+
+Flink is designed to process continuous streams of data at a lightning fast pace. This short guide
+will show you how to download the latest stable version of Flink, install, and run it. You will 
+also run an example Flink job and view it in the web UI. 
 
-  Since you are currently looking at the latest `SNAPSHOT`
-  version of the documentation, all version references below will not work.
-  Please switch the documentation to the latest released version via the release picker which you
-  find on the left side below the menu.
+
+## Downloading Flink
+
+{{< hint info >}}
+__Note:__ Flink is also available as a [Docker image](https://hub.docker.com/_/flink).
 {{< /hint >}}
-{{< /unstable >}}
 
-Follow these few steps to download the latest stable versions and get started.
+Flink runs on all UNIX-like environments, i.e. Linux, Mac OS X, and Cygwin (for Windows). You need 
+to have __Java 8 or 11__ installed. To check the Java version installed, type in your terminal: 
 
-## Step 1: Download
+```bash
+$ java -version
+```
 
-To be able to run Flink, the only requirement is to have a working __Java 8 or 11__ installation.
-You can check the correct installation of Java by issuing the following command:
+Next, [download the latest binary release](https://flink.apache.org/downloads.html) of Flink, 
+then extract the archive: 
 
 ```bash
-java -version
+$ tar -xzf flink-*.tgz
 ```
 
-[Download](https://flink.apache.org/downloads.html) the {{< version >}} release and un-tar it. 
+## Browsing the project directory
+
+Navigate to the extracted directory and list the contents by issuing:
 
 ```bash
-$ tar -xzf flink-{{< version >}}-bin-scala{{< scala_version >}}.tgz
-$ cd flink-{{< version >}}-bin-scala{{< scala_version >}}
+$ cd flink-* && ls -l
 ```
 
-## Step 2: Start a Cluster
+You should see something like:
+
+![project directory](projectdirectory.png)
+
+For now, you may want to note that:
+- __bin/__ directory contains the `flink` binary as well as several bash scripts that manage various jobs and tasks
+- __conf/__ directory contains configuration files, including `flink-conf.yaml`
+- __examples/__ directory contains sample applications that can be used as is with Flink
 
-Flink ships with a single bash script to start a local cluster.
+
+## Starting and stopping a local cluster
+
+To start a local cluster, run the bash script that comes with Flink:
 
 ```bash
 $ ./bin/start-cluster.sh
-Starting cluster.
-Starting standalonesession daemon on host.
-Starting taskexecutor daemon on host.
 ```
 
-## Step 3: Submit a Job
+You should see an output like this:
+
+![output](output.png)
```

Comment:
@sjwiesman The images don't seem to be rendering for me.  Does it work for you?

Reply:
You caught me, I didn't build it locally 😅 

The images should be under `static/fig`. There is a subfolder for `learn-flink`. Everything in the 'static' folder is put at the root of the URL when building. The image path will then be:

```raw
![output](/fig/learn-flink/installation/output.png)
```